airqo-platform/AirQo-frontend
# Analytics download csv

- What does this PR do?
Download data as CSV.
- What JIRA task is related to this PR?
No JIRA card.
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/analytics/` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run local` to test out the server.
- Which end points should are ready for testing?
No endpoints
